transcom/mymove
# B 19506 validation code feature flag int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/808731)

## Warning
Please approve the [Flipt pull request](https://github.com/transcom/milmove-feature-flags/pull/121) prior to this one

## Summary

Introduces validation code feature flag for MilMove. The Flipt pull request can be found [here](https://github.com/transcom/milmove-feature-flags/pull/121).

### How to test

1. Pull the branch down
2. Run `make feature_flag_docker`
3. Access `http://localhost:9080/` 
4. `validation_code_required` should exist

## Screenshots
![image](https://github.com/transcom/mymove/assets/136814362/769e18b0-648e-4733-bdbf-d1fd103dca19)
